Sources/App/configure.swift

```diff
@@ -16,21 +16,27 @@ private let jiraProjects = [
     "ETA"   : 17369, // Engagement Tribe - Activate
     "ETR"   : 17372, // Engagement Tribe - Retain
     "GW"    : 16949, // Triage UI
+    "HC"    : 17251, // HealthCheck (next-gen)
     // TODO: [IOSP-101/IOSP-147] Re-enable IDM once this board have migrated away from NextGen board
     // "IDM"   : 16903, // Identity Platform / Identity Management
     "IDP"   : 17228, // Identity Access Management
     "IOSP"  : 17263, // iOS Native Apps Platform
+    "MC"    : 17006, // Member Communications (next-gen)
     "MS"    : 17233, // Monitor
     "MON"   : 10103, // HealthCheck
     "NRX"   : 16911, // GP@Hand Registrations (ex Enrolment and Integrity)
     "PAR"   : 17098, // Partnerships
     "PRO"   : 16980, // Professional Services
     "PRSCR" : 16840, // Prescriptions
+    "REFER" : 17244, // Referrals (next-gen)
     "SDK"   : 16975, // SDK
+    "SDKS"  : 17288, // SDK UI (next-gen)
+    "TC"    : 17353, // Triage Comprehension (next-gen)
     "TEL"   : 16857, // Telus
     // TODO: [IOSP-101/268] Re-enable TEN once this board have migrated away from NextGen board
     // "TEN"   : 16955,
```

Comment:
I think at that point we could as well remove these commented lines and mark IOSP-268 as "Won't Fix", because I think this board is obsolete anyway?

Reply:
Done 👍